src/Orders/EmailAccounts/Responses/CreateResponse.php

```diff
@@ -33,6 +34,11 @@ public function __construct(array $attributes = [])
     {
         parent::__construct($attributes);
 
+        // Not all responses contain a status it would seem, for example get business email.
+        if (!$this->wasSuccessful()) {
```

Comment:
Should be a space after the “not” operator, i.e. `if (! $this->wasSuccessful())`.